api-spec/redirect/redirect-api.yaml

```diff
@@ -0,0 +1,218 @@
+openapi: 3.0.0
```

Comment:
is this openapi used for client generation? could we insert a folder in the file path that indicates this? for example `client/openapi/*.yaml` or `openapi/client/*.yaml`

Reply:
Yes I like it. Can we leave api-spec as root folder or maybe is preferable to move it out?

Reply:
Fixed with 99dd8836a5ea433dadf7c77a54c5de8b9a0475e6